boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -178,20 +175,20 @@ Security components
 Programming and Debugging
 *************************
 
-nRF5340 Application MCU supports the Armv8m Security Extension.
-Applications build for the nrf5340pdk_nrf5340_cpuapp board by default
+nRF5340 application core supports the Armv8m Security Extension.
+Applications build for the nrf5340dk_nrf5340_cpuapp board by default
 boot in the Secure state.
 
-nRF5340 Network MCU does not support the Armv8m Security Extension.
-nRF5340 IDAU may configure bus accesses by the nRF5340 Network MCU
+nRF5340 network core does not support the Armv8m Security Extension.
```

Comment:
```suggestion
nRF5340 network core does not support the Armv8-M Security Extension.
```

Reply:
updated. This is now consistent, at least within the nRF5340 Docs.
In general we need to see how to spell ARM :)